src/main/java/uk/gov/hmcts/reform/divorce/orchestration/tasks/SendPetitionerCoRespondentRespondedNotificationEmail.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.divorce.orchestration.tasks;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.divorce.orchestration.framework.workflow.task.Task;
+import uk.gov.hmcts.reform.divorce.orchestration.framework.workflow.task.TaskContext;
+import uk.gov.hmcts.reform.divorce.orchestration.service.EmailService;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.D_8_CASE_REFERENCE;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.D_8_PETITIONER_EMAIL;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.D_8_PETITIONER_FIRST_NAME;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.D_8_PETITIONER_LAST_NAME;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.NOTIFICATION_ADDRESSEE_FIRST_NAME_KEY;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.NOTIFICATION_ADDRESSEE_LAST_NAME_KEY;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.NOTIFICATION_REFERENCE_KEY;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.RECEIVED_AOS_FROM_RESP;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.RESP_WILL_DEFEND_DIVORCE;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.YES_VALUE;
+
+@Component
+public class SendPetitionerCoRespondentRespondedNotificationEmail implements Task<Map<String, Object>> {
+
+    private final EmailService emailService;
+
+    @Autowired
+    public SendPetitionerCoRespondentRespondedNotificationEmail(EmailService emailService) {
+        this.emailService = emailService;
+    }
+
+    @Override
+    public Map<String, Object> execute(TaskContext context, Map<String, Object> caseData) {
+
+        String petitionerEmail = (String) caseData.get(D_8_PETITIONER_EMAIL);
+
+        if (StringUtils.isNotBlank(petitionerEmail)) {
+            Map<String, String> templateVars = new HashMap<>();
+            
+            templateVars.put(NOTIFICATION_REFERENCE_KEY, (String) caseData.get(D_8_CASE_REFERENCE));
+            templateVars.put(NOTIFICATION_ADDRESSEE_FIRST_NAME_KEY, (String) caseData.get(D_8_PETITIONER_FIRST_NAME));
+            templateVars.put(NOTIFICATION_ADDRESSEE_LAST_NAME_KEY, (String) caseData.get(D_8_PETITIONER_LAST_NAME));
+
+            // Only send email to Petitioner when the respondent has not defended the case yet
+            if (!isRespondentDefendedCase(caseData)) {
+                if (isRespondentRespondedCase(caseData)) {
```

Comment:
This condition has to be before the other one, if the respondent has not respond you will never know he  defends or not the divorce

Reply:
I think its easier like this, since basically, I don't want any emails when its a defended case. So if the respondent hasn't answered yet, its a null value and implicitly not defended. In which case I do want to send an email.
Since the other check is if they have or haven't responded.

The alternative is something like:
                if (isRespondentRespondedCase(caseData) && !isRespondentDefendedCase(caseData)) {
                    emailService.sendPetitionerEmailCoRespondentRespondWithAosNoDefend(petitionerEmail, templateVars);
                } else if (!isRespondentDefendedCase(caseData)) {
                    emailService.sendPetitionerEmailCoRespondentRespondWithAosNotStarted(petitionerEmail, templateVars);
                }

Reply:
When the respondent has not responded will never be defending
```suggestion
                if (isRespondentRespondedCase(caseData)) {
                     if ( !isRespondentDefendedCase(caseData)) {
                         emailService.sendPetitionerEmailCoRespondentRespondWithAosNoDefend(petitionerEmail, templateVars);
                     }
                } else {
                     emailService.sendPetitionerEmailCoRespondentRespondWithAosNotStarted(petitionerEmail, templateVars);
                }
```

Reply:
Done